sheet.go

```diff
@@ -103,6 +115,66 @@ func (s *Sheet) SetColWidth(startcol, endcol int, width float64) error {
 	return nil
 }
 
+// When merging cells, the cell may be the 'original' or the 'covered'.
+// First, figure out which cells are merge starting points. Then create
+// the necessary cells underlying the merge area.
+// Then go through all the underlying cells and apply the appropriate
+// border, based on the original cell.
+func (s *Sheet) handleMerged() {
+	merged := make(map[string]*Cell)
+
+	for r, row := range s.Rows {
+		for c, cell := range row.Cells {
+			if cell.HMerge > 0 || cell.VMerge > 0 {
+				coord := fmt.Sprintf("%s%d", numericToLetters(c), r+1)
+				merged[coord] = cell
+			}
+		}
+	}
+
+	// Now go through the map and create the ghost cells
```

Comment:
I suck at comments (trying to become better), but maybe something along the lines of:

```golang
// This loop iterates over all cells that should be merged and applied the correct
// borders to them depending on their position. If any cells required by the merge
// are missing, they will be allocated by s.Cell().
```

?